### ISSUE TITLE
PouchDB not using sqlite_plugin for adapter: websql inside angularjs controller/service

### ISSUE THREAD

**Issue by virtualit786** [pouchdb/pouchdb#4981]
Hi @nolanlawson,

I am using PouchDB in an ionic application as shown in the following code snippets:-

```
var db = PouchDB('UserDB.db', {adapter: 'websql'});
db.info().then(console.log.bind(console));
```
OR
 
If I put the above both code in a angularjs controller or service, it gives the following output which shows its not using the sqlite_plugin:-

> 
 {"doc_count":0,"update_seq":0,**"sqlite_plugin":false**,"websql_encoding":"UTF-8","db_name":"UserDB.db","auto_compaction":false,"adapter":"websql"}

But if I put the above code in a angularjs run code block, then it gives the following output which shows its using the sqlite_plugin:-

> 
{"doc_count":0,"update_seq":0,**"sqlite_plugin":true,**"websql_encoding":"UTF-8","db_name":"UserDB.db","auto_compaction":false,"adapter":"websql"}

Result is same even if I use 
`var db = pouchDB('UserDB.db', {adapter: 'websql'});`
I am using the Cordova-sqlcipher-adapter plugin https://github.com/litehelpers/Cordova-sqlcipher-adapter

Any idea on whats going wrong ?

**Comment by virtualit786** [comment]
Hi again @nolanlawson 
I saw you mentioning a related bug that you fixed in release 5.31. For the info, I have tried with 5.1.0 and then also took the latest code snapshot dated 18Mar2016 which says  5.3.2-prerelease in package.json. And its not working in either of these versions!
Can you please check !

**Comment by nolanlawson** [comment]
Please do this check:

```html
<script src="sqlite_plugin.js"></script>
<script>
console.log('is the SQLite plugin available? ' + window.sqlitePlugin);
</script>
<script src="pouchdb.js"></script>
```

The SQLite Plugin needs to be loaded and available before PouchDB is loaded. Then it should work. If that's not the case, please reopen this issue, but it's usually a problem with the ordering of script tags or the fact that the SQLite Plugin is not correctly installed. Please also see this if you need a working example: https://github.com/nolanlawson/pouchdb-cordova-hello-world-with-sqlite-plugin or https://github.com/nolanlawson/pouchdb-ionic
